### ISSUE TITLE
Could you clarify the input 2d and output 3d data?

### ISSUE THREAD

**Issue by ShuangjunLiu** [ArashHosseini/3d-pose-baseline#5]
Hello, since the data is not available. 
Could you clarify the 2d and 3d pose data input to the network?  
I believe the 3d is root center coordinate in camera space.  
Is 2d data in original image space in pixel unit?   
Am I correct? 

If that is the case, this model is trained on H36M specific camera configuration.  
When applied on images in the wild, is that still valid considering different resolution and camera parameters?

**Comment by kaurrachneet6** [comment]
I have a similar question. If I have computed the 2D coordinates in pixels using my own camera and then used your code to compute 3D coordinates in cm (in camera coordinates) without tuning the model, i.e. I used your code directly to infer 3D points (because in my data, actual 3D coordinates are unknown and hence, I cannot tune the model). Would this method still give me correct 3D coordinates in cm? Or will it not, because my camera is different than what was used for training the model. Thanks.
